.github/workflows/build.yml

```diff
@@ -129,7 +129,8 @@ jobs:
 
       - name: Install Playwright
         working-directory: apps/wing-console/console/app
-        run: pnpm exec playwright install --with-deps
+        run: sudo rm /etc/apt/sources.list.d/microsoft-prod.list
+          pnpm exec playwright install --with-deps
```

Comment:
Looks like there's still a problem with the `--with-deps` attribute:
https://github.com/winglang/wing/actions/runs/8815708146/job/24198200226?pr=6311